cmd/ketch/testdata/app-log/1.output

```diff
@@ -0,0 +1,16 @@
+hello-web-1 0
```

Comment:
The timestamp is an essential piece of information that should be included in every log entry. It's no use to know that something happened without knowing when it happened.
https://www.scalyr.com/blog/log-formatting-best-practices-readable/#:~:text=The%20timestamp%20is%20an%20essential,without%20knowing%20when%20it%20happened.

Reply:
it's optional, you can enable timestamps with `--timestamps`